### DISCUSSION TITLE
Need add features like to create a REST API

### DISCUSSION THREAD

**Discussion by sw-tt-jaydeepcharadva** [ExpressionEngine/ExpressionEngine#3724] [Feature Requests]
Need to add features like create a REST API for manage member using another web portal.

**Comment by sw-tt-jaydeepcharadva** [comment]
Hi @intoeetive,

Is there any update to add this feature in EE?

**Reply by intoeetive** [reply]
@sw-tt-jaydeepcharadva I believe there are several add-ons that might fit your need in the add-on store. If you are looking for a specific implementation advice, you may get some community help in ExpressionEngine Slack
